conf/app.routes

```diff
@@ -1,6 +1,7 @@
 # microservice specific routes
 
-GET       /                                               uk.gov.hmrc.payetaxcalculatorfrontend.controllers.IndexController.index()
+GET       /                                               uk.gov.hmrc.payetaxcalculatorfrontend.controllers.IndexController.legacy()
+GET       /new                                            uk.gov.hmrc.payetaxcalculatorfrontend.controllers.IndexController.index()
```

Comment:
Not sure if we should have `/new` and `/` pointing to old or rather `/` pointing to new and `/old` pointing to old. What do people think?